build-farm/platform-specific-configurations/linux.sh

```diff
@@ -88,12 +88,18 @@ if [ "${JAVA_TO_BUILD}" == "${JDK11_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JD
       export PATH=/opt/rh/devtoolset-7/root/usr/bin:$PATH
       [ -r /opt/rh/devtoolset-7/root/usr/bin/gcc ] && export CC=/opt/rh/devtoolset-7/root/usr/bin/gcc
       [ -r /opt/rh/devtoolset-7/root/usr/bin/g++ ] && export CXX=/opt/rh/devtoolset-7/root/usr/bin/g++
-    elif [ -r /usr/local/gcc/bin ]; then
+    elif [ -r /usr/local/gcc/bin/gcc-7.3 ]; then
```

Comment:
this logic looks like we will fail for anything that is not gcc-7.3 or 7.4. Is that correct?

Reply:
Yes - previously the situation was worse - if it detected `/usr/local/gcc/bin` then it would fail if `gcc-7.3` wasn't inside so this is safer